.github/workflows/apply-clang-format.yml

```diff
@@ -0,0 +1,17 @@
+name: Apply clang-format to PR
+
+on:
+  pull_request:
+    types: [labeled]
+
+jobs:
+  clang-format:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+         fetch-depth: 0
+    - uses: InsightSoftwareConsortium/ITKApplyClangFormatAction@master
+      with:
+        github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Is `apply-clang-format.yml` still needed? Git warnings have been fixed in Github actions as of [https://github.com/actions/checkout/pull/762](https://github.com/actions/checkout/pull/762)

Reply:
I will force-push without `apply-clang-format.yml`.